Latest pekko milestone

pjfanning opened the pull request:
relates to #1352 
seeing if the CI build picks up the issue

ivantopo commented:
Hey @pjfanning just FYI I just merged a PR that fixed the dependency issues from the Apache client/cxf instrumentation. You might want to update your branch to build again :)

pjfanning commented:
Thanks @ivantopo - I updated this branch to take in latest master changes.

pekko-protobuf jar is no longer needed and has been removed from Pekko 1.1 - causing the latest issues for this PR.

I have removed that dependency in #1353. Would it be possible to merge that PR?

pjfanning commented:
closing due to #1362
